nftables_test.go

```diff
@@ -16,18 +16,25 @@ package nftables_test
 
 import (
 	"bytes"
+	"flag"
 	"fmt"
 	"net"
+	"runtime"
 	"strings"
 	"testing"
 
 	"github.com/google/nftables"
 	"github.com/google/nftables/binaryutil"
 	"github.com/google/nftables/expr"
 	"github.com/mdlayher/netlink"
+	"github.com/vishvananda/netns"
 	"golang.org/x/sys/unix"
 )
 
+var (
+	enableSysTests = flag.Bool("run_system_tests", false, "Run tests that operate against the live kernel")
```

Comment:
Now that we’re running these in a throwaway network namespace, is there any point in having the flag? Are there any adverse side-effects to protect developers from? If no, please remove.

Reply:
Yes, this is still too dangerous to let it run without active acknowledgment (though more IPTables fault than NFTables).

Here is one of many scenarios where we would fuck up the system:
1. User uses iptables on their system
2. User runs our tests
3. Our tests test the NAT functionality, causing the `nf_nat` kernel module to be loaded, along with a bunch others.
4. The load of `nf_nat` silently uses the (single) hook in the kernel for NAT functionality, silently breaking IPTables.

Reply:
I just tested without the `Flushruleset` bit and I can see some test remnants from _outside_ the network namespace - suggesting nftables is not fully namespace aware.

Reply:
Nevermind - forgot a single method of `Dial` in the refactor. Works fine in a network namespace.

Reply:
Good point!

In that case, can you make sure that travis actually runs these tests, though?